### ISSUE TITLE
Carthage integration is broken on Release 6.4.1

### ISSUE THREAD

**Issue by carlosfernandez** [Iterable/swift-sdk#525]
The build log reads: 
```
Touch /Users/carlos/Library/Caches/org.carthage.CarthageKit/DerivedData/13.2.1_13C100/swift-sdk/6.4.1/Build/Products/Release-iphonesimulator/IterableAppExtensions.framework (in target 'notification-extension' from project 'swift-sdk')
    cd /Users/carlos/Documents/iOS_Projects/work/merryfield-ios/Carthage/Checkouts/swift-sdk
    /usr/bin/touch -c /Users/carlos/Library/Caches/org.carthage.CarthageKit/DerivedData/13.2.1_13C100/swift-sdk/6.4.1/Build/Products/Release-iphonesimulator/IterableAppExtensions.framework

** BUILD FAILED **


The following build commands failed:
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler (in target 'swift-sdk' from project 'swift-sdk')
	CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler (in target 'swift-sdk' from project 'swift-sdk')
	CompileSwiftSources normal i386 com.apple.xcode.tools.swift.compiler (in target 'swift-sdk' from project 'swift-sdk')
	CompileSwift normal i386 (in target 'swift-sdk' from project 'swift-sdk')
```

Xcode 13.2.1
Carthage 0.38

Macbook Pro M1

**Comment by tapashmajumder** [comment]
Hi. Does it run on a real device, i.e., not a simulator?

**Comment by tapashmajumder** [comment]
Should be fixed with version 6.4.2. Please open a ticket with your CSM if you are still seeing issues.
